TcUnit-Runner/AutomationInterface.cs

```diff
@@ -98,5 +100,14 @@ public void StartRestartTwinCAT()
         {
             sysManager.StartRestartTwinCAT();
         }
+        public bool TcPinnedVersion
+        {
+            get
+            {
+                return sysManager.TcVersionFixed;
```

Comment:
Hmm... why do we need the rest of the code (with XML-parsing and everything, GetPinnedVersionXML) if the sysmanager already provides this?

Reply:
Now I come back why XML parser (only needs ITcSysManager10) and not `SysManger.TcVersionFixed ` (requires ITcSysManager13) 

The ITcSysManager interface only works if the project is loaded.
But the check has to take place before loading. Therefore I could not use `SysManger.TcVersionFixed.`

If I tried to load a pinned project with a version which is not available, the RemoteManger automatically ( `silentMode = true`) 
loaded the last version. (no matter if pinned or not)
and automatically upgraded my pinned version to the last version. 

At the end of the unit test my project was still pinned, but on the wrong version

Reply:
Does this mean that we will never use the SysManager.TcVersionFixed?

Reply:
I have to go to version 4020.0 and check the interface ITcSysManager13.

I assume that it does not work. This would mean we cannot use SysManager.TcVersionFixed from the interface ITcSysManager13